nautobot/extras/datasources/git.py

```diff
@@ -188,12 +190,14 @@ def git_repository_diff_origin_and_local(repository_pk, request, job_result_pk,
 
 def get_repo_from_url_to_path_and_from_branch(repository_record):
     """Returns the from_url, to_path and from_branch of a Git Repo
-    :return (list): [
+    :return (namedtuple): (
         from_url: git repo url with token or user if available,
         to_path: path to location of git repo on local machine
         from_branch: current git repo branch
-    ]
+    )
     """
+    GitRepo = namedtuple("GitRepo", ["from_url", "to_path", "from_branch"])
```

Comment:
- [ ] Same as with `GitJobResult`, can you please move this to a module-level global and revise the docstring for `get_repo_from_url_to_path_and_from_branch`? Thanks!
- [ ] Also, just for the sake of readability, can you also please rename `GitRepo` to something like `GitRepoInfo` since this isn't actually a Git repo, but just information about one?